pkgs/applications/networking/instant-messengers/webcord/default.nix

```diff
@@ -56,36 +53,34 @@ buildNpmPackage rec {
     install -Dm644 sources/assets/icons/app.png $out/share/icons/hicolor/256x256/apps/webcord.png
 
     # Add xdg-utils to path via suffix, per PR #181171
-    makeWrapper '${electron_25}/bin/electron' $out/bin/webcord \
-      --prefix LD_LIBRARY_PATH : ${libPath}:$out/opt/webcord \
+    makeWrapper '${lib.getExe electron_25}' $out/bin/webcord \
+      --prefix LD_LIBRARY_PATH : ${lib.makeLibraryPath [ libpulseaudio pipewire ]}:$out/opt/webcord \
```

Comment:
I'm not sure if this change to remove `libPath` is super necessary, but it's fine ig.

Reply:
Yeah was trying to make it consistent with the bin path